libpod/image/search.go

```diff
@@ -147,6 +153,45 @@ func searchImageInRegistry(term string, registry string, options SearchOptions)
 	// every types.SystemContext, and to compute the value just once in one
 	// place.
 	sc.SystemRegistriesConfPath = sysreg.SystemRegistriesConfPath()
+	if options.ListTags {
+		dockerPrefix := fmt.Sprintf("%s://", docker.Transport.Name())
+		imageRef, err := alltransports.ParseImageName(fmt.Sprintf("%s/%s", registry, term))
+		if err == nil && imageRef.Transport().Name() != docker.Transport.Name() {
+			logrus.Errorf("reference %q must be a docker reference", term)
```

Comment:
@rhatdan got a less Docker way to say this?  Perhaps:
```suggestion
			logrus.Errorf("reference %q must be a valid container image reference", term)
```

Reply:
I think "docker" is more accurate in containers-transports(5) terms as all of the transports refer to a container image.

Reply:
@vrothberg sometimes I reall don't like it when you're right...  Let's leave it then, thx for the follow up.